File: MY PROGRAMS/LB/JAVA/NUMBER/a1q4.java
/*	4. Accept one number and check whether is is divisible by 5 or not.	*/




import java.util.*;

class Assignment1
{
	public boolean Boolean(int ino)
	{
		if((ino%5)==0)	
		{
			return(true);
		}
		else
		{
			return(false);
		}
	}
}

class a1q4
{
	public static void main(String arg[])
	{
		Scanner sobj=new Scanner(System.in);

		int ivalue=0;
		boolean bret=false;

		System.out.println("Enter the no.: ");
		ivalue=sobj.nextInt();

		Assignment1 aobj=new Assignment1();

		bret=aobj.Boolean(ivalue);

		if(bret==true)
		{
			System.out.println("YES , the given no. is divisible by 5.");
		}
		else
		{
			System.out.println("NO , the given no. is not divisible by 5.");
		}
	}
}

File: MY PROGRAMS/LOGIC BUILDING-13 ASSIGNMENTS/a1q4.c
/* Accept one number and check whether is is divisible by 5 or not. */


#include<stdio.h>

#define YES 1
#define NO 0

typedef int BOOL;

BOOL Divide(int ino)
{
	if((ino%5)==0)
	{
		return(YES);
	}
	else
	{
		return(NO);
	}
}

int main()
{
	int ivalue=0;
	BOOL bret=NO;

	printf("Enter the no.:");
	scanf("%d",&ivalue);

	bret=Divide(ivalue);

	if(bret==YES)
	{
		printf("No. is divided by 5.");
	}
	else
	{
		printf("No. is not divided by 5.");
	}

	return(0);
}

